molvis/molvispython/python/molvisdata.py

```diff
@@ -0,0 +1,155 @@
+import inviwopy as ivw
+
+default = {'rgb': [0, 0, 0], 'radius': 3.00}
+
+atomColors = {
```

Comment:
again? we already have these in base?

Reply:
you mean `misc/vasp/python/atomdata.py` ? That is not base.
https://github.com/inviwo/modules/blob/master/misc/vasp/python/atomdata.py

Reply:
No I ment the these molvis/molvisbase/include/inviwo/molvisbase/util/molvisutils.h

Reply:
now properly exposed lookup functions in `atomielement.h` to python